src/applications/personalization/profile360/tests/actions/paymentInformation.unit.spec.js

```diff
@@ -41,37 +43,62 @@ const teardown = () => {
 };
 
 describe('actions/paymentInformation', () => {
-  beforeEach(setup);
-  afterEach(teardown);
+  describe('when global `ga` is set up correctly', () => {
+    beforeEach(() => setup({ mockGA: true }));
+    afterEach(teardown);
 
-  it('calls fetch and dispatches FETCH_PAYMENT_INFORMATION_SUCCESS', async () => {
-    const actionCreator = paymentInformationActions.fetchPaymentInformation();
-    const dispatch = sinon.spy();
+    it('calls fetch and dispatches FETCH_PAYMENT_INFORMATION_SUCCESS', async () => {
+      const actionCreator = paymentInformationActions.fetchPaymentInformation();
+      const dispatch = sinon.spy();
 
-    await actionCreator(dispatch);
+      await actionCreator(dispatch);
 
-    expect(dispatch.called).to.be.true;
-    expect(dispatch.firstCall.args[0].type).to.be.equal(
-      paymentInformationActions.PAYMENT_INFORMATION_FETCH_SUCCEEDED,
-    );
-    expect(global.fetch.called).to.be.true;
-  });
+      expect(dispatch.called).to.be.true;
+      expect(dispatch.firstCall.args[0].type).to.be.equal(
+        paymentInformationActions.PAYMENT_INFORMATION_FETCH_SUCCEEDED,
+      );
+      expect(global.fetch.called).to.be.true;
+    });
+
+    it('calls fetch and dispatches SAVE_PAYMENT_INFORMATION', async () => {
+      const actionCreator = paymentInformationActions.savePaymentInformation({
+        data: 'value',
+      });
+      const dispatch = sinon.spy();
+
+      await actionCreator(dispatch);
 
-  it('calls fetch and dispatches SAVE_PAYMENT_INFORMATION', async () => {
-    const actionCreator = paymentInformationActions.savePaymentInformation({
-      data: 'value',
+      expect(global.fetch.called).to.be.true;
+      expect(dispatch.calledTwice).to.be.true;
+      expect(dispatch.firstCall.args[0].type).to.be.equal(
+        paymentInformationActions.PAYMENT_INFORMATION_SAVE_STARTED,
+      );
+      expect(dispatch.secondCall.args[0].type).to.be.equal(
+        paymentInformationActions.PAYMENT_INFORMATION_SAVE_SUCCEEDED,
+      );
     });
-    const dispatch = sinon.spy();
+  });
+
+  describe('when `ga` is not set up correctly', () => {
+    beforeEach(() => setup({ mockGA: false }));
+    afterEach(teardown);
 
-    await actionCreator(dispatch);
+    it('calls fetch and dispatches SAVE_PAYMENT_INFORMATION', async () => {
```

Comment:
I confirmed that this test fails as expected if the access to `ga` is not wrapped in a `try...catch`